### DISCUSSION TITLE
Is there an option to change the timestamp format?

### DISCUSSION THREAD

**Discussion by spenserblack** [lowlighter/metrics#564] [Ideas]
I didn't see the option, but wanted to make sure I wasn't missing anything. If the option doesn't currently exist, this discussion should probably be moved to the "Ideas" category.

I would like to be able to change the timestamp from `dd/mm/yyyy` format to `yyyy/mm/dd` format, as I believe this format is less confusing across regions.

**Comment by lowlighter** [comment]
I'm on mobile so I cannot link the code part responsible for this, but it's somewhere around the formatters in source/app/utils.mjs

I think dates are using the en-GB format so we could change it.

I don't remember where exactly there are dates, but if it doesn't mess up display I would go with short month notation so it's even clearer (e.g 12th Oct. 2021), though it's just a personal opinion

**Reply by spenserblack** [reply]
OK, I'm going to move this discussion to "Ideas" since it sounds like the option doesn't exist as an input.

**Reply by lowlighter** [reply]
This was changed since to the above suggestion (short month notation)

I don't think we'll introduce an option to configure date format as it brings less value than maintaining work imho.
If it is that much important, I guess it's possible to achieve the same with [`extras_js`](https://github.com/lowlighter/metrics/tree/master/source/plugins/core#%EF%B8%8F-custom-javascript-scripting) option, use a custom template or a fork

**Reply by spenserblack** [reply]
Sounds good to me. I think my issue of "confusing format across regions" is resolved by the short month notation.

But it's a bit biased towards Europe and the Americas. A use-case that I can think of for changing format would be a user from an Asian country like China or Japan might prefer "2022年2月8日" over "Feb. 8th, 2022". Personally, I'm good with leaving it as it is, but just something you might want to consider.

**Reply by lowlighter** [reply]
Yeah but I guess in such cases you'd just want to translate the whole svg, it would feel kind of weird to just have the date stand out from the rest

Translations are actually a nice idea of for custom templates though 🤔 
It'd fit well with config_presets too I guess

**Reply by spenserblack** [reply]
>in such cases you'd just want to translate the whole svg

Good point, this can definitely be considered "closed" then :+1:

>Translations are actually a nice idea of for custom templates though

Cool idea!
